schema/library.raml

```diff
@@ -394,12 +398,6 @@ types:
       - geoaggregator
       - latitude
       - longitude
-  API_VariableScale:
-    type: string
-    enum:
-      - log
-      - log2
-      - ln
```

Comment:
Looks like a few things got lost in the merge with Ryan's DV code. I don't think this should be removed in addition to some of the other changes in this file.

I think you can just checkout the latest EDA common and re-run `./gradlew gen-jaxrs` to fix this up.